### PR TITLE
Response

### PR DESCRIPTION
**Pull request by noobiwankenoobi**

### CODE REVIEW

**Comment by noobiwankenoobi** [comment]
Comfort: 4
Clarity: 4

I got hung up on something and didn't finish every question.
I used the cheatsheet I created as well as Google for this diagnostic.

File: diagnostic.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
  <!-- Answer Starts Here -->
-
+Absolute path, because it specifies the entire filepath.
```
Comment (Jcornmanhomonoff) [inline diagnostic.md:58]: Correct, but more specifically we know this because the first `/`represents the root of the system

File: diagnostic.md [hunk 2]
```diff
@@ -71,7 +71,7 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 If we are in the `project` directory and use `cd planning`, is a relative or absolute path being referenced? How do you know?
 
  <!-- Answer Starts Here -->
-
+i dunno
```
Comment (Jcornmanhomonoff) [inline diagnostic.md:74]: Relative path, planning is referenced with respect to the current working directory, project.

File: diagnostic.md [hunk 3]
```diff
@@ -71,7 +71,7 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
...
  <!-- Answer Ends Here -->
 
 3. Now suppose that we have an image file living inside our project. Would we refer to it with an absolute or relative path? Why?
```
Comment (Jcornmanhomonoff) [inline diagnostic.md:77]: Relative, the image file lives inside our current working directory, project ; as a result, an absolute path is not needed to access it.

**Comment by Jcornmanhomonoff** [comment]
5/7! Great job! Keep up the awesome work! 🦄 👍

**Pull request closed without merging**